src/components/ListItem/ListItem.styles.ts

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+
+import { cVar, sizes } from '@/styles'
+
+import { ListItemSizes } from '.'
+import { SvgActionCheck } from '../_icons'
+
+const getContainerPadding = (size: ListItemSizes) => {
+  switch (size) {
+    case 'large':
+      return css`
+        padding: ${sizes(3)} ${sizes(4)};
+      `
+    case 'medium':
+      return css`
+        padding: ${sizes(2)} ${sizes(4)};
+      `
+    case 'small':
+      return css`
+        padding: ${sizes(1)} ${sizes(4)};
+      `
+  }
+}
+type ContainerProps = { size: ListItemSizes; hasNodeStart: boolean; disabled?: boolean }
+const disabledStyles = css`
+  opacity: 0.25;
+  cursor: not-allowed;
+`
+const interactiveStyles = css`
+  &:focus,
+  &:hover {
+    background-color: ${cVar('colorBackgroundAlpha')};
+  }
+
+  &:active {
+    background-color: ${cVar('colorBackgroundMutedAlpha')};
+  }
+`
+export const Container = styled.div<ContainerProps>`
+  display: grid;
+  grid-gap: ${sizes(3)};
+  grid-auto-flow: column;
+  grid-template-columns: ${({ hasNodeStart }) => (hasNodeStart ? 'auto 1fr' : '1fr')};
+  align-items: center;
+  user-select: none;
```

Comment:
This component should have also different default background if Im not mistaken. 
```
background: var(--color-background-muted);
```
Design: 
![image](https://user-images.githubusercontent.com/51168865/144422424-8e4bbb5f-b2c4-47bc-855e-6ebbd3cf88ef.png)

Reply:
https://github.com/Joystream/atlas/pull/1701#discussion_r761041470